packages/jazz-tools/src/tools/tests/coMap.record.test.ts

```diff
@@ -194,6 +194,15 @@ describe("CoMap.Record", async () => {
         $jazz: expect.objectContaining({ id: me.$jazz.id }),
       });
     });
+
+    test("getEdits() keys should return deleted keys", () => {
+      const Person = co.record(z.string(), z.string());
+      const person = Person.create({ name: "John" });
+      person.$jazz.set("name", "Jane");
+      person.$jazz.delete("name");
+
+      expect(Object.keys(person.$jazz.getEdits())).toEqual(["name"]);
+    });
```

Comment:
Could you add a test to validate what's into the deleted edits?

Reply:
Done in 9cc2770ba78c5c6b627762c9058cb06fda89cb77

The ability to review edits to deleted items was already available, but there wasn't a way to find those keys.

Reply:
This change also affects keys that are explicitly set to undefined, right? If so, mind adding a test for that case as well?